src/typing/customResponse.ts

```diff
@@ -0,0 +1,6 @@
+export type CustomResponse = {
```

Comment:
responseObject: {} is object or Array? This code can break because Array is a Object from Array Class in Globals ... Look on Line 20, VenueServices. 

Another name for responseObject is data.  but type is Array of User or Venue..
realErrorMessage we dont know... you can add as unknown type.

Reply:
Fixed this up, changed payload to data to follow convention.